iclr2017/python/superuser-assign-reviewers.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/python
+
+###############################################################################
+# Not to spec
+###############################################################################
+
+## Import statements
+import argparse
+import csv
+import getpass
+import sys
+import re
+from openreview import *
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('assignments', help="either (1) a csv file containing reviewer assignments or (2) a string of the format '<email_address>,<paper#>' e.g. 'reviewer@cs.umass.edu,23'")
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+if args.username!=None and args.password!=None:
+    client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
+else:
+    client = Client(baseurl=args.baseurl)
+baseurl = client.baseurl
+
+submissions = client.get_notes(invitation='ICLR.cc/2017/conference/-/submission')
+
+def single_assignment_valid(s):
+    try:    
+        areachair = s.split(',')[0]
+        paper_number = s.split(',')[1]
+
+        try: 
+            int(paper_number)
+        except ValueError:
+            return False
+
+        if not '@' in areachair:
+            return False
+
+        return True
+    except IndexError:
+        return False
+
+def assign_reviewer(reviewer,paper_number):
+    notes = [note for note in submissions if str(note.number)==str(paper_number)]
```

Comment:
you could use the get notes by invitation and paper number. or save the submissions in a dictionary where the key is the paper number, so you don't need to iterate the collection for each assingments